modules/base/tool_test.go

```diff
@@ -0,0 +1,104 @@
+package base
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestEncodeMD5(t *testing.T) {
+	if checksum := EncodeMD5("foobar"); checksum != "3858f62230ac3c915f300c664312c63f" {
+		t.Errorf("got the wrong md5sum for string foobar: %s", checksum)
+	}
+
+}
+
+func TestEncodeSha1(t *testing.T) {
+	if checksum := EncodeSha1("foobar"); checksum != "8843d7f92416211de9ebb963ff4ce28125932878" {
+		t.Errorf("got the wrong sha1sum for string foobar: %s", checksum)
+	}
+}
+
+func TestShortSha(t *testing.T) {
+	if result := ShortSha("veryverylong"); result != "veryverylo" {
```

Comment:
I thought this was a hash-function too. But if you look closer it only shortens a string. Pretty bad function naming. But I don't want to change the API with the tests. That should be a different PR.